README.md

```diff
@@ -149,6 +149,14 @@ Meeting notes are held in [this doc](https://docs.google.com/document/d/1oHpisis
 
 You can also join us on [gitter](https://gitter.im/open-telemetry/opentelemetry-java)
 
+#### Java Auto-Instrumentation
+
+Separate but related to the "Java SDK SIG", the "Java Auto-Instrumentation SIG" meets every other week on Thursday at 9 AM Pacific Time [TODO create calendar and add link]. For more context, see [the auto-instrumentation OTEP](https://github.com/open-telemetry/oteps/blob/master/text/0001-telemetry-without-manual-instrumentation.md).
```

Comment:
I created a cal event: https://calendar.google.com/event?action=TEMPLATE&tmeid=MXExNnRoNTlkZWppMGR2NWZndDNxZWI1dmYgZ29vZ2xlLmNvbV9iNzllM2U5MGo3YmJzYTJuMnA1YW41bGY2MEBn&tmsrc=google.com_b79e3e90j7bbsa2n2p5an5lf60%40group.calendar.google.com

Reply:
Can you change event to start next week (Thu, Oct 3), and to repeat every 2 weeks?

Reply:
Done, sorry. By the way, you should have edit access to the meeting (or that was my intention).